agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/messaging/EndpointsListService.java

```diff
@@ -0,0 +1,12 @@
+package com.dke.data.agrirouter.api.service.messaging;
+
+import agrirouter.request.payload.account.Endpoints;
+import com.dke.data.agrirouter.api.dto.onboard.OnboardingResponse;
+import com.dke.data.agrirouter.api.service.parameters.EndpointsListParameters;
+
+public interface EndpointsListService extends MessagingService<EndpointsListParameters> {
+
+  String requestFullListFiltered(OnboardingResponse onboardingResponse);
```

Comment:
What does "full" in this context mean? A filtered list can not be the "full" list in my point of view.

Reply:
Reasonable input; Renamed to requestFullListFilteredByAppliedRoutings